src/main/tables/91_quick_launch_favorites.sql

```diff
@@ -0,0 +1,12 @@
+SET search_path = public, pg_catalog;
+
+CREATE TABLE quick_launch_favorites (
+  -- Primary key
+  id uuid NOT NULL,
+
+  -- Foreign key into the quick_launches table
+  quick_launch_id uuid NOT NULL,
+
+  -- Foreign key into the users table
+  user_id uuid NOT NULL
+);
```

Comment:
Another option would be to use the metadata service to mark something as a favorite. I haven't convinced myself that one way is better than the other yet. Apps in the DE are still marked as favorites using the `Favorites` category. Files and folders are marked as favorites using the metadata service. I think this option is fine, but I thought I should mention the metadata service in case we wanted to move toward a consistent implementation of favorites.

One caveat is that it appears that using the metadata service might require a little more work initially because the endpoints are specific to data items. I took a quick look at the code, however, and it does look like the favorites feature in the metadata service was intended to be used for all types of resources.

Reply:
Sarah and I discussed this in person and the conclusion we agreed on is that we shouldn't modify the metadata service as part of the quick launches feature. We can always make that change as its own work task.